fineract-provider/src/main/java/org/apache/fineract/cob/loan/StayedLockedLoansTasklet.java

```diff
@@ -33,16 +39,25 @@
 @RequiredArgsConstructor
 public class StayedLockedLoansTasklet implements Tasklet {
 
-    private final LoanAccountLockRepository loanAccountLockRepository;
     private final BusinessEventNotifierService businessEventNotifierService;
+    private final LoanRepository loanRepository;
 
     @Override
     public RepeatStatus execute(StepContribution contribution, ChunkContext chunkContext) throws Exception {
-        List<LoanAccountLock> loanAccountLocks = loanAccountLockRepository.findAll();
-        if (!loanAccountLocks.isEmpty()) {
-            List<Long> loanIds = loanAccountLocks.stream().map(LoanAccountLock::getLoanId).toList();
-            businessEventNotifierService.notifyPostBusinessEvent(new LoanAccountsStayedLockedBusinessEvent(loanIds));
+        LoanAccountsStayedLockedBusinessEvent.Data lockedLoanAccounts = buildLoanAccountData();
+        if (!lockedLoanAccounts.getLoanAccounts().getLoanAccounts().isEmpty()) {
```

Comment:
why the double getLoanAccounts()?

Reply:
used unneccessary wrapper class, removed it.